backend/pick-git/src/main/java/com/woowacourse/pickgit/user/domain/User.java

```diff
@@ -30,64 +33,83 @@
     private GithubProfile githubProfile;
 
     @Embedded
-    private Followers followers = new Followers();
+    private Followers followers;
 
     @Embedded
-    private Followings followings = new Followings();
+    private Followings followings;
 
     @Embedded
-    private Posts posts = new Posts();
+    private Posts posts;
 
     protected User() {
     }
 
-    public User(
-        Long id,
-        BasicProfile basicProfile,
-        GithubProfile githubProfile) {
-        this.id = id;
-        this.basicProfile = basicProfile;
-        this.githubProfile = githubProfile;
+    public User(BasicProfile basicProfile, GithubProfile githubProfile) {
+        this(null, basicProfile, githubProfile);
+    }
+
+    public User(Long id, BasicProfile basicProfile, GithubProfile githubProfile) {
+        this(id, basicProfile, githubProfile, new Followers(new ArrayList<>()),
+            new Followings(new ArrayList<>()), new Posts(new ArrayList<>()));
```

Comment:
```suggestion
        this(
            id, 
            basicProfile, 
            githubProfile, 
            new Followers(new ArrayList<>()),
            new Followings(new ArrayList<>()), 
            new Posts(new ArrayList<>())
        );
```
생성자에 어떤 데이터가 들어가는지 확실히 보여주려면 This도 이 컨벤션을 따르는게 좋을듯 합니당

Reply:
User 도메인 로직 변경과 함께 반영할게요!